core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/DrmSessionErrorForwarder.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.drm.DefaultDrmSessionManager;
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.exoplayer.playererror.DrmInitiatingError;
+import com.novoda.noplayer.listeners.ErrorListeners;
+
+class DrmSessionErrorForwarder implements DefaultDrmSessionManager.EventListener {
+
+    private final Player player;
+    private final ErrorListeners errorListeners;
+
+    DrmSessionErrorForwarder(Player player, ErrorListeners errorListeners) {
+        this.player = player;
+        this.errorListeners = errorListeners;
+    }
+
+    @Override
+    public void onDrmKeysLoaded() {
+        // TODO: Are we interested?
```

Comment:
currently all the `*forwarders` have a todo mentioning if we should track them, would prefer to do them all at the same time